gocd/bci_repo_publish.py

```diff
@@ -126,11 +140,16 @@ def run(self, version, token=None):
             return
 
         # If the last published build is less than a day old, don't publish
-        newest_published_mtime = max([int(pkg['published_mtime']) for pkg in packages])
-        published_build_age_hours = int(time.time() - newest_published_mtime) // (60 * 60)
-        if published_build_age_hours < 24:
-            self.logger.info('Current published build less than a day old '
-                             f'({published_build_age_hours}h).')
+        oldest_published_mtime = min([int(pkg['published_mtime']) for pkg in packages])
+        oldest_published_version = min([pkg['published_version'] for pkg in packages])
+        published_build_age_hours = int(time.time() - oldest_published_mtime) // (60 * 60)
+        if int(float(built_version)) != int(float(oldest_published_version)):
```

Comment:
Those are versions, not floats...

Reply:
well, they seem to be in the format <CI_CNT>.<BCNT>, but I can make it use version comparison logic as well